lms/static/js/groups/views/cohort_discussions_inline.js

```diff
@@ -17,10 +17,11 @@
                     },
 
                     render: function () {
-                        var alwaysCohortInlineDiscussions = this.cohortSettings.get('always_cohort_inline_discussions');
+                        var alwaysCohortInlineDiscussions = this.cohortSettings.get('always_cohort_inline_discussions'),
+                            inline_discussions = this.model.get('inline_discussions');
 
                         this.$('.cohort-inline-discussions-nav').html(this.template({
-                            inlineDiscussionTopics: this.getInlineDiscussionsHtml(this.model.get('inline_discussions')),
+                            inlineDiscussionTopicsHtml: this.getInlineDiscussionsHtml(inline_discussions),
```

Comment:
@robrap and/or @andy-armstrong What is the best practice in this sort of case? I renamed the variable to indicate that it is HTML, but should I wrap it in HtmlUtils.HTML as well?

And in the template where it is used, should I use HtmlUtils.ensureHtml?